modules/nf-core/pureclip/main.nf

```diff
@@ -0,0 +1,69 @@
+process PURECLIP {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::pureclip=1.3.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/pureclip:1.3.1--0':
+        'biocontainers/pureclip:1.3.1--0' }"
+
+    input:
+    tuple val(meta), path(ipbam), path(controlbam)
+    tuple val(meta), path(ipbai), path(controlbai)
+    path genome_fasta
```

Comment:
have a look at some other modules, i thought the rule now was that all inputs like this now needed meta - `tuple val(meta2), path(genome_fasta)`

Reply:
can you point me to a module like that? havent seen it in any other modules ive looked at